SQL.js

```diff
@@ -138,6 +151,8 @@ class SqlStatement {
 
     return this
   }
+
+  get [SqlStatementSymbol] () { return true }
```

Comment:
standard won't let me use `[SqlStatementSymbol] = true`, I guess I could put it in the constructor...

Reply:
also see https://github.com/nearform/sql/pull/52#discussion_r627618510